cmd/networkcmd/clean.go

```diff
@@ -34,4 +49,17 @@ func clean(cmd *cobra.Command, args []string) {
 	} else {
 		ux.Logger.PrintToUser("Process terminated.")
 	}
+
+	if hard {
+		ux.Logger.PrintToUser("hard clean requested via flag, removing all downloaded avalanchego and plugin binaries")
+		binDir := filepath.Join(app.GetBaseDir(), constants.AvalancheCliBinDir)
+		cleanBins(binDir)
+	}
+}
+
+func cleanBins(dir string) {
+	if err := os.RemoveAll(dir); err != nil {
+		ux.Logger.PrintToUser("Removal failed: %s", err)
+	}
+	ux.Logger.PrintToUser("All existing binaries removed. To continue using the tool, install latest binaries again.")
```

Comment:
Confused by the last sentence. The user may think he needs to install binaries by itself, but really the tool is going to make that for him on the next command execution. Probably just drop the last sentence?

Reply:
After the fact I agree